terraform/modules/daos_server/templates/daos_startup_script.tftpl

```diff
@@ -51,8 +51,8 @@ $${DAOS_DIR}/cert_gen/sm_get_ca.sh "${daos_ca_secret_id}" "server"
 systemctl enable daos_server
 systemctl start daos_server
 
-#give time for dmg commands to work 
-sleep 5
+systemctl enable daos_agent
```

Comment:
@lsitkiew  There can be a race here.   Please add the sleep back in so DMG always works.

Reply:
I didn't get any problem here. Everything is working correctly without that sleep phase for me. I will check this.